yarn.lock

```diff
@@ -1756,11 +1756,6 @@ alphanum-sort@^1.0.0:
   resolved "https://registry.yarnpkg.com/alphanum-sort/-/alphanum-sort-1.0.2.tgz#97a1119649b211ad33691d9f9f486a8ec9fbe0a3"
```

Comment:
Do you have `package.json` changes uncommited? Just wonderig how `yarn.lock` got modified without them 😄

Reply:
:thinking:  maybe it's some rebasing magic, or I think I ran `yarn install` a couple of times

Reply:
Running `yarn install` should not make changes to the lock file without changes in `package.json` 🤔

Reply:
The changes seem to be due to some cleanup, as I only see stuff being removed. Maybe I have deleted (I think I did once) the `node_modules` folder and so **yarn** had to reinstall all the dependencies, which might have changed since last time

Reply:
If dependencies don't change and the lock file is intact, running `yarn install` should not change anything (only install the packages into `node_modules`). I think this file can be reverted to the version `master` has.

If you want to experiment a bit, I'd be interested in the results of:
1. revert `yarn.lock` changes
2. `yarn install`
3. are there any changes to `yarn.lock`?

Reply:
> are there any changes to yarn.lock?

Nope 🤔